src/Cake.Issues/IReadOnlyDictionaryExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Cake.Issues
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public static class IReadOnlyDictionaryExtensions
```

Comment:
@pascalberger I can't make it internal because it is used by `IssueChecker` in the Testing project aswell. Any better ideas?

Reply:
@janniksam Adding an `InternalsVisibleTo` attribute [here](https://github.com/cake-contrib/Cake.Issues/blob/develop/src/Cake.Issues/Properties/AssemblyInfo.cs#L14) to the `Cake.Issues.Testing` project would solve it.

But I'm not sure having the extension is the best way to go, since both usages are quite different. In the `IssueComparer` you're only interested in the boolean result, while in the `IssueChecker` you also want to know what's the difference is to show a appropriate message. Since it's basically a very simple extension I would suggest to get rid of it and implement it directly suitable for the specific needs.